python/mxnet/runtime.py

```diff
@@ -27,17 +27,27 @@
 
 class Feature(ctypes.Structure):
     """
-    Compile time feature description
+    Compile time feature description, member fields: `name` and `enabled`.
     """
     _fields_ = [
         ("_name", ctypes.c_char_p),
-        ("enabled", ctypes.c_bool)
+        ("_enabled", ctypes.c_bool)
     ]
 
     @property
     def name(self):
+        """
+        feature name
+        """
         return self._name.decode()
 
+    @property
+    def enabled(self):
+        """
+        True if MXNet was compiled with the given compile-time feature
```

Comment:
```suggestion
        True if MXNet was compiled with the given compile-time feature.
```
Also, there's no example provided for this function.